### PR TITLE
fix #846 : Server-Sent Events (SSE) Support

### LINKED ISSUE #846: [WEB Connector] Server-Sent Events (SSE) Support

### PR DESCRIPTION
**Pull request by elebescond**

### CODE REVIEW

**Comment by vsct-jburet** [comment]
This is ok for me. Do you need this merged in 20.3 release, or can it wait a next build release?

**Comment by elebescond** [comment]
No ... I think it will not work with multiple instances of a bot.

I will try this approach
https://docs.mongodb.com/manual/core/tailable-cursors

**Comment by vsct-jburet** [comment]
Please use https://docs.mongodb.com/manual/changeStreams/ - they are already used in tock - look at https://doc.tock.ai/tock/dokka/tock/ai.tock.shared/com.mongodb.reactivestreams.client.-mongo-collection/index.html @elebescond

File: bot/connector-web/src/main/kotlin/channel/ChannelMongoDAO.kt [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    )
+    private val database: MongoDatabase by injector.instance(
+        MONGO_DATABASE
+    )
+    var asyncWebChannelResponseCol: com.mongodb.reactivestreams.client.MongoCollection<ChannelEvent>
+    var webChannelResponseCol: MongoCollection<ChannelEvent>
+
+    init {
+        try {
+            database
```
Comment (elebescond) [inline bot/connector-web/src/main/kotlin/channel/ChannelMongoDAO.kt:42]: Not terrible ... any suggestions are welcome (ping @vsct-jburet )

**Pull request merged**